modules/ROOT/nav.adoc

```diff
@@ -33,6 +33,7 @@
   *** xref:dataweave-type-reuse-json-schema.adoc[Reusing Types from a JSON Schema]
   *** xref:dataweave-type-reuse-xml-schema.adoc[Reusing an XML Schema]
   *** xref:dataweave-type-reuse-java-classes.adoc[Reusing Types from Java Classes]
+  *** xref:dataweave-type-reuse-avro-schema.adoc[Reusing Types from an Avro Schema]
```

Comment:
Just making sure you want to hard-code link text here...

Reply:
I guess Daniela followed the existing convention for the other type loaders